assignments/tasks_test.go

```diff
@@ -125,7 +125,7 @@ func TestHandleTasks(t *testing.T) {
 	accessToken := scm.GetAccessToken(t)
 
 	logger := log.Zap(false).Sugar()
-	scm, err := scm.NewSCMClient(logger, "github", accessToken)
+	scm, err := scm.NewSCMClient(logger, scm.GetTestClient(t, qfTestOrg), "github", accessToken)
```

Comment:
Should this use instead (since this is a `_test.go` file):
```go
qtest.TestSCMClient(ctx, t, qfTestOrg, "github", accessToken)
```

Reply:
Should `TestSCMClient` accept a `logger` as input or have a way to enable it internally for local logging... during testing